setools/boolquery.py

```diff
@@ -61,6 +61,10 @@ def results(self):
         self._match_name_debug(self.log)
         self.log.debug("Default: {0.default}".format(self))
 
+        # try to translate old boolean names (not with regexps)
+        if self.name and isinstance(self.name, str):
+            self.name = self.policy.check_boolean_sub(self.name)
```

Comment:
I've been thinking about this more and I think the boolean name substitution should go in the cli programs, and the substitution should only apply if no policy is specified on the command line. Otherwise cases where you are analyzing a policy from another system may be impacted. Say I take a policy from Gentoo system and analyze it on my Fedora system. The Fedora name substitutions do not necessarily apply to the Gentoo policy I am querying.